src/components/AnalysisContent/index.js

```diff
@@ -62,60 +73,99 @@ function AnalysisContent({
+                data={{ title, source: documentSource }}
+              />
+            </ThemeContainer>,
+            indicator
+          );
+        }
+        return indicator;
+      });
+    }
+
+    const hurumapCharts = Array.from(
+      document.querySelectorAll('div[data-chart-id]')
+    );
+    if (hurumapCharts && hurumapCharts.length > 0) {
+      hurumapCharts.map(chart => {
+        const chartId = chart.attributes['data-chart-id'].value;
+        const geoId = chart.attributes['data-geo-type'].value;
+        ReactDOM.render(
+          <ApolloContainer>
+            <ThemeContainer>
+              <HURUmapChart geoId={geoId} chartId={chartId} />
+            </ThemeContainer>
+          </ApolloContainer>,
+          chart
+        );
+        return chart;
+      });
+    }
+  }, [takwimu.country.name, topicIndex]);
+
   const [carouselItemIndex, setCarouselItemIndex] = useState(
-    content.body[topicIndex].type === 'carousel_topic' ? 0 : -1
+    content.topics[topicIndex].type === 'carousel_topic' ? 0 : -1
   );
   useEffect(() => {
     setCarouselItemIndex(
-      content.body[topicIndex].type === 'carousel_topic' ? 0 : -1
+      content.topics[topicIndex].type === 'carousel_topic' ? 0 : -1
     );
-  }, [content.body, topicIndex]);
+  }, [content.topics, topicIndex]);
 
-  const [id, setId] = useState(`${content.id}-${topicIndex}`);
-  useEffect(() => {
-    if (`${content.id}-${topicIndex}` !== id) {
-      setId(`${content.id}-${topicIndex}`);
-    }
-  }, [content.id, topicIndex, id]);
+  // const [id, setId] = useState(`${content.id}-${topicIndex}`);
+  // useEffect(() => {
+  //   if (`${content.id}-${topicIndex}` !== id) {
+  //     setId(`${content.id}-${topicIndex}`);
+  //   }
+  // }, [content.id, topicIndex, id]);
 
-  useEffect(() => {
-    if (document.getElementsByClassName('flourish-embed')) {
-      const script = document.createElement('script');
-      const oldScript = document.getElementById('flourish-script');
-      if (oldScript) {
-        oldScript.remove();
-      }
-
-      window.FlourishLoaded = false;
-      script.id = 'flourish-script';
-      script.src = 'https://public.flourish.studio/resources/embed.js';
-      document.body.appendChild(script);
-    }
-  }, [id, carouselItemIndex]);
+  // useEffect(() => {
+  //   if (document.getElementsByClassName('flourish-embed')) {
+  //     const script = document.createElement('script');
+  //     const oldScript = document.getElementById('flourish-script');
+  //     if (oldScript) {
+  //       oldScript.remove();
+  //     }
+
+  //     window.FlourishLoaded = false;
+  //     script.id = 'flourish-script';
+  //     script.src = 'https://public.flourish.studio/resources/embed.js';
+  //     document.body.appendChild(script);
+  //   }
+  // }, [topicIndex]);
 
   const showContent = index => () => {
     onChange(index);
   };
+  const profileNavigation = 'Other topics in';
+  const readNext = 'Read next...';
```

Comment:
Weren't we moving these to the dashboard?

Reply:
@kilemensi yes, this will be on another PR together with the "where to next.." links